iTjuana/itj-library
# Get account data from server

### Description of the Change


Use database and session data to populate account page.
If not logged in, no info will be displayed, instead will appear a button to login.

### Verification Process


1. Run the UI
2. Go to the account page
3. Log in
4. See data

### Applicable Issues


- Closes #86 

### Release Notes


N/A
